tests/ship_streamer.cpp

```diff
@@ -128,12 +130,41 @@ int main(int argc, char* argv[]) {
          eosio::check(result_doucment[1]["head"].IsObject(),                                 "'head' is not an object");
          eosio::check(result_doucment[1]["head"].HasMember("block_num"),                     "'head' does not contain 'block_num'");
          eosio::check(result_doucment[1]["head"]["block_num"].IsUint(),                      "'head.block_num' isn't a number");
+         eosio::check(result_doucment[1]["head"].HasMember("block_id"),                      "'head' does not contain 'block_id'");
+         eosio::check(result_doucment[1]["head"]["block_id"].IsString(),                     "'head.block_num' isn't a string");
```

Comment:
Should this be `result_document`?

Reply:
lol, Took me awhile to figure out what you were saying. I'll fix the spelling.